Logger/Logger.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace Klaviyo\Reclaim\Logger;
+
+class Logger extends \Monolog\Logger
```

Comment:
should this be called KlaviyoLogger?

Reply:
It can be, it is under the \Klaviyo\Reclaim\Logger namespace already so it feels a bit redundant to me.

Reply:
Since it would look like this wherever we use it:
\Klaviyo\Reclaim\Logger\KlaviyoLogger